nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/pom.xml

```diff
@@ -83,6 +83,10 @@
                     <groupId>com.google.code.findbugs</groupId>
                     <artifactId>jsr305</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
This should be moved to a `jigsaw` profile to limit the exclusion to builds done with a JDK version > 8.